### PR TITLE
[Merged by Bors] - chore: fix spelling mistakes in set theory files

### PR DESCRIPTION
**Pull request by euprunin**

### CODE REVIEW

File: Mathlib/SetTheory/Game/Nim.lean [hunk 1]
```diff
@@ -65,11 +65,11 @@ theorem moveLeft_nim_hEq (o : Ordinal) :
 theorem moveRight_nim_hEq (o : Ordinal) :
     HEq (nim o).moveRight fun i : o.toType => nim ((enumIsoToType o).symm i) := by rw [nim_def]; rfl
 
-/-- Turns an ordinal less than `o` into a left move for `nim o` and viceversa. -/
+/-- Turns an ordinal less than `o` into a left move for `nim o` and vice-versa. -/
```
Comment (grunweg) [inline Mathlib/SetTheory/Game/Nim.lean:68]:
```suggestion
/-- Turns an ordinal less than `o` into a left move for `nim o` and vice versa. -/

File: Mathlib/SetTheory/Game/Nim.lean [hunk 2]
```diff
@@ -65,11 +65,11 @@ theorem moveLeft_nim_hEq (o : Ordinal) :
...
 noncomputable def toLeftMovesNim {o : Ordinal} : Set.Iio o ≃ (nim o).LeftMoves :=
   (enumIsoToType o).toEquiv.trans (Equiv.cast (leftMoves_nim o).symm)
 
-/-- Turns an ordinal less than `o` into a right move for `nim o` and viceversa. -/
+/-- Turns an ordinal less than `o` into a right move for `nim o` and vice-versa. -/
```
Comment (grunweg) [inline Mathlib/SetTheory/Game/Nim.lean:72]:
```suggestion
/-- Turns an ordinal less than `o` into a right move for `nim o` and vice versa. -/

**Review by grunweg — COMMENTED** [review]
Thanks! Please adjust the two words I mentioned, but generally looks good to me.
maintainer merge

**Comment by b-mehta** [comment]
bors merge

**Pull request closed without merging**